sdk/rust/oak_tests/src/lib.rs

```diff
@@ -504,25 +504,10 @@ where
     Q: protobuf::Message,
 {
     // Put the request in a GrpcRequest wrapper and serialize into a message.
-    let mut grpc_req = oak::proto::grpc_encap::GrpcRequest::new();
-    grpc_req.set_method_name(method_name.to_string());
-    let mut any = protobuf::well_known_types::Any::new();
-    if let Err(e) = req.write_to_writer(&mut any.value) {
-        warn!("failed to serialize gRPC request: {}", e);
-        return Err(oak::grpc::build_status(
-            oak::grpc::Code::INTERNAL,
-            "failed to serialize gRPC request",
-        ));
-    }
-    grpc_req.set_req_msg(any);
-    grpc_req.set_last(true);
-    let mut req_msg = OakMessage {
-        data: Vec::new(),
+    let req_msg = OakMessage {
+        data: oak::grpc::encap_request(req, method_name).expect("failed to serialize GrpcRequest"),
```

Comment:
Some of this is already implemented here:

https://github.com/project-oak/oak/blob/04785c2231d404fcdbe2a740eb5c08696096c9ed/sdk/rust/oak/src/grpc/mod.rs#L39-L45